charts/nri-kube-events/templates/deployment.yaml

```diff
@@ -120,8 +120,9 @@ spec:
           emptyDir: {}
         - name: tmpfs-tmp
           emptyDir: {}
-      {{- if .Values.priorityClassName }}
-      priorityClassName: {{ .Values.priorityClassName }}
+      {{- with include "common.priorityClassName" . }}
+      priorityClassName:
+        {{- . | nindent 8 -}}
```

Comment:
My bad, updated in 43f840e0a3f4367aca4975e51336a2108742e894

Reply:
This is still templated a map. It is a string 😅 

```suggestion
      priorityClassName: {{ . }}
```

I you have doubts, take a look at [CHART-TEMPLATE](https://github.com/newrelic/helm-charts/blob/master/library/CHART-TEMPLATE/templates/integration-deployment.yaml#L29-L31).

Reply:
I said nothing. You said it was solved on other commit you force-pushed so I left the comment just in case it was not properly solved.

It is solved.